src/Raven.Server/Integrations/PostgreSQL/PgSession.cs

```diff
@@ -210,6 +211,23 @@ public async Task Run()
             {
                 // Terminate silently
             }
+            catch (QueryParser.ParseException e)
+            {
+                if (Logger.IsInfoEnabled)
+                    Logger.Info("Invalid RQL query", e);
```

Comment:
Should include the actual query as well,no?

Reply:
I thought the query is included in the parse exception itself. Anyway at this level it's difficult to actually get the RQL query but I see we have it logged in `PgQuery`:

https://github.com/ravendb/ravendb/pull/12989/files#diff-988f2be98095d88fdd397fa81b60afcced725d67cd6668718759b093accb0503R58-R59